descriptors/ModuleDescriptor-template.json

```diff
@@ -416,9 +416,71 @@
           "permissionsRequired": ["patron-action-session-storage.expired-session-patron-ids.collection.get"]
         }
       ]
+    },
+    {
+      "id": "check-in-operation-storage",
+      "version": "0.1",
+      "handlers": [
+        {
+          "methods": [
+            "GET"
+          ],
+          "pathPattern": "/check-in-operation-storage",
```

Comment:
Typically, with the storage APIs, they use a structure like `/item-storage/items` could these be adapted to fit that kind of structure, e.g. `/check-in-operation-storage/check-ins`?